main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative './app'
+
+def launch
+  app = App.new
+  app.start_app
```

Comment:
- [ ] Just as the previous reviewer highlighted, the app crashes with a response about the start_app method you called on the app. It's not running the app at the moment. Kindly use `app.run` as the run method starts the app successfully. Please work on that change. 👍🏽

<img width="870" alt="Screenshot 2022-12-24 at 5 47 24 PM" src="https://user-images.githubusercontent.com/50941074/209446636-060904a4-14a5-4471-8eeb-6439c52b454c.png">